spec/controllers/admin/tracks_controller_spec.rb

```diff
@@ -409,31 +406,80 @@
     end
 
     context 'has start_date and end_date' do
-      it_behaves_like 'fails to accept', true, true, false
+      before :each do
+        self_organized_track.start_date = Date.current
+        self_organized_track.end_date = Date.current
```

Comment:
Using the predefined track with all attributes, you can be simply removing the ones you do **not** want, which is going to reduce the lines of our tests (that's good because we have less to read and understand).